kinetic/distribution.yaml

```
@@ -3841,7 +3841,11 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/HebiRobotics/hebi_cpp_api_ros-release.git
-      version: 2.0.1-1
+      version: 2.0.2-0
+    source:
+      type: git
+      url: https://github.com/HebiRobotics/hebi_cpp_api_ros.git
+      version: 2.0.2
```

The version here should be the development branch to check out not a tag.
```suggestion
      version: master
```